### DISCUSSION TITLE
checkout

### DISCUSSION THREAD

**Discussion by niagaku909** [woocommerce/woocommerce#52250] [Checkout Flow]
![Gambar WhatsApp 2024-10-23 pukul 09 31 36_9079f037](https://github.com/user-attachments/assets/18eed088-2387-42af-a1e7-248143daf560)

always write like that,,,,need addres,,,i have make a good addres and not wrong,,,,so how i can fix that

**Comment by juliaamosova** [comment]
Closing as duplicate of https://github.com/woocommerce/woocommerce/discussions/52249.
